src/quoridor/controller/gamecontroller.go

```diff
@@ -5,50 +5,93 @@ import (
 
 	"quoridor/game"
 	"quoridor/storage"
-
-	"github.com/lithammer/shortuuid"
 )
 
+type Party struct {
+	game game.Game
+	players map[string]int
+}
+
+func (p Party) isReady() bool {
+	return len(p.players) == len(p.game.Pawns)
+}
+
+func getParty(id string) (Party, error) {
+	p, found := storage.Get(id)
+	if !found {
+		return Party{}, errors.New("The game does not exist")
+	}
+	return p.(Party), nil
+}
+
+func checkPlayer(p Party, playerID string) error {
```

Comment:
You should rename it IMO.

What "check player" means ? Please find a more signifiant name like checkPlayerCanPlay, isPlayerTurn, ensurePlayerCanPlay...